src/views/user/LoginView.vue

```diff
@@ -216,6 +256,13 @@
             })
         },
 
+        mounted() {
+            let recaptchaScript = document.createElement('script')
```

Comment:
could be a `const` (but don't correct this now, sometime soon we'll have auto linting for this)

Reply:
👍